utils/swift_build_support/swift_build_support/targets.py

```diff
@@ -68,6 +68,18 @@ def contains(self, target_name):
                 return True
         return False
 
+    def swift_flags(self, args):
+        """
+        Swift compiler flags for a platform, useful for cross-compiling
+        """
+        return ''
+
+    def cmake_options(self, args):
+        """
+        CMake flags to build for a platform, useful for cross-compiling
+        """
+        return ''
```

Comment:
Since empty strings are passed here for every cross-compilation host but Android, this pull is very safe.